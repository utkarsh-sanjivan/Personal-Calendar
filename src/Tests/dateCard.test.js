import React from 'react';
import { create } from "react-test-renderer";
import Date from './../Components/DateCard/index';

test('Test Today Date Component Component', () => {
    const component = create(<Date 
        onDateClick={() => {}}
        onSeeMoreClick={() => {}}
        onEventClick={() => {}}
        formattedDate={''}
        isActive={true}
        events={[]}
        date={'10 August 2020'}
        textClassName=''
    />);
  expect(component.toJSON()).toMatchSnapshot()
});

test('Test Active Date Component Component', () => {
    const component = create(<Date 
        onDateClick={() => {}}
        onSeeMoreClick={() => {}}
        onEventClick={() => {}}
        formattedDate={''}
        isActive={true}
        events={[]}
        date={'10 August 2020'}
        textClassName=''
    />);
  expect(component.toJSON()).toMatchSnapshot()
});

test('Test Inactive Date Component Component', () => {
    const component = create(<Date 
        onDateClick={() => {}}
        onSeeMoreClick={() => {}}
        onEventClick={() => {}}
        formattedDate={''}
        isActive={true}
        events={[]}
        date={'10 August 2020'}
        textClassName=''
    />);
  expect(component.toJSON()).toMatchSnapshot()
});