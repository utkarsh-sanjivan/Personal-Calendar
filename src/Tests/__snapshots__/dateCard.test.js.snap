// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Active Date Component Component 1`] = `
<div
  className="card-container"
>
  <div
    className=""
  >
    <div
      className="date-text"
      onClick={[Function]}
    >
      
    </div>
  </div>
</div>
`;

exports[`Test Inactive Date Component Component 1`] = `
<div
  className="card-container"
>
  <div
    className=""
  >
    <div
      className="date-text"
      onClick={[Function]}
    >
      
    </div>
  </div>
</div>
`;

exports[`Test Today Date Component Component 1`] = `
<div
  className="card-container"
>
  <div
    className=""
  >
    <div
      className="date-text"
      onClick={[Function]}
    >
      
    </div>
  </div>
</div>
`;
